blog/_posts/2024-02-26-scala-3.4.0-and-3.3.2-released.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: blog-detail
+post-type: blog
+by: Paweł Marks, VirtusLab
+title: Scala 3.4.0 and 3.3.2 LTS released!
+---
+We are thrilled to announce the release of two versions of Scala 3: the first version in the 3.4 minor line, and a patch version in the Long Term Support line.
```

Comment:
is there a fancy 3.4 banner image?

Reply:
maybe put a call out to the community to suggest a mascot for this post